roles/edpm_nova/tasks/configure.yml

```diff
@@ -77,18 +62,18 @@
   check_mode: false
   delegate_to: localhost
 
-- name: Flatten configmaps into {{ edpm_nova_config_dest }}
+- name: Flatten configmaps into {{ edpm_nova_config_sync_src }}
+  become: true
```

Comment:
see also https://ansible.readthedocs.io/projects/lint/rules/no-same-owner/#problematic-code

perhaps we need to use 
```
owner: false
group: false
```
to not change nova for root?

Reply:
done

Reply:
i mind using become: true less of an issue in that AEE although were not required yes i prefer to avoid it.

i belive you require become here because

/var/lib/openstack/configs/custom/nova/ is not owned by the use in the AEE container  perhaps we should use something like '{{ lookup('ansible.builtin.env', 'HOME') }}/staging'

that would allow you to drop become here.

Reply:
so this become true shoudl not be needed now right

"{{ lookup('ansible.builtin.env', 'HOME') }}/staging/ is owned by the current user